data/json/recipes/recipe_others.json

```diff
@@ -3796,7 +3819,9 @@
   "difficulty" : 2,
```

Comment:
It's difficulty 2 to put a bunch of lab gear in a bag?  This seems like it should be 0.